From f57329c6fc2d0787f3d020d029bce96a5ea0399b Mon Sep 17 00:00:00 2001
From: Janani Sankara Babu <jananis37@gmail.com>
Date: Thu, 21 Sep 2017 11:09:55 +0530
Subject: [PATCH] staging:rtl8188eu:hal Fix wrong comparison to False
Git-commit: f57329c6fc2d0787f3d020d029bce96a5ea0399b
Patch-mainline: v4.15-rc1
References: FATE#326906

This patch solves the warning "Using comparison to false is error prone"

Signed-off-by: Janani Sankara Babu <jananis37@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/staging/rtl8188eu/hal/odm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/rtl8188eu/hal/odm.c b/drivers/staging/rtl8188eu/hal/odm.c
index ec8aae76bf40..001d6267b56e 100644
--- a/drivers/staging/rtl8188eu/hal/odm.c
+++ b/drivers/staging/rtl8188eu/hal/odm.c
@@ -382,7 +382,7 @@ void odm_DIG(struct odm_dm_struct *pDM_Odm)
 	}
 
 	/* add by Neil Chen to avoid PSD is processing */
-	if (pDM_Odm->bDMInitialGainEnable == false) {
+	if (!pDM_Odm->bDMInitialGainEnable) {
 		ODM_RT_TRACE(pDM_Odm, ODM_COMP_DIG, ODM_DBG_LOUD, ("odm_DIG() Return: PSD is Processing\n"));
 		return;
 	}
-- 
2.19.1

